recipes/threeml/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "threeml" %}
+{% set version = "2.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/threeML-{{ version }}.tar.gz
+  sha256: 3725b801d27b9e667fca0db7dfd9f309d9c9bf00bf187c3b15d8988ce9ac07f4
+  patches: 
+    - pandas.patch
```

Comment:
Use patch for pandas 1.4: https://github.com/threeML/threeML/commit/a28d93befc7771c41f18a7d2e406809cd0da156b